package.json

```diff
@@ -55,7 +55,7 @@
     "phantomjs-prebuilt": "^2.1.3",
     "sinon": "^1.17.3",
     "sinon-chai": "^2.8.0",
-    "vue": "^1.0.19",
+    "vue": "^1.0.24",
```

Comment:
This change is not required and won't really have any effect. I think it is safe to leave out.